src/com_tjvendors/admin/controllers/payout.php

```diff
@@ -35,6 +35,8 @@ public function __construct()
 			$this->client = $this->input->get('client', '');
 		}
 
+		$this->text_prefix = 'COM_TJVENDORS_PAYOUTS';
```

Comment:
@deepa-g Why we need this?

Reply:
To handle success message via core joomla code we need to set this